cli/data/enti/download.py

```diff
@@ -4,18 +4,87 @@
 #
 # Copyright (C) 2022 Giacomo Tesio <giacomo@tesio.it>
 # Copyright (C) 2022 Leonardo Canello <leonardocanello@protonmail.com>
+# Copyright (C) 2022 Andrea Foletto <andrea@yaaaw.it>
 #
 # MonitoraPA is a hack. You can use it according to the terms and
 # conditions of the Hacking License (see LICENSE.txt)
 
 import sys
 import requests
-import commons
-
-outDir = commons.computeOutDir(sys.argv)
-url = 'https://indicepa.gov.it/ipa-dati/datastore/dump/d09adf99-dc10-4349-8c53-27b1e5aa97b6?bom=True&format=tsv'
-r = requests.get(url, allow_redirects=True)
-result = r.content.replace(
-    b'HTTPS:SISTEMAAMBIENTELUCCA.IT', b'HTTPS://SISTEMAAMBIENTELUCCA.IT')
-open("%s/enti.tsv" % outDir, 'wb').write(result)
-print("%s/enti.tsv" % outDir)
+import datetime
+import os
+import shutil
+
+def computeOutDir(argv):
+
+    if not os.getcwd() == "/":
+        print("[ Error ] Please run cli/ scripts from the root directory")
+        sys.exit(1)
+
+    if not os.path.isdir("out"):
+        os.mkdir("out")
+
+    dirName = f"out/{datetime.datetime.utcnow().strftime('%Y-%m-%d')}"
+
+    if not os.path.isdir(dirName):
+        os.mkdir(dirName)
+        print(f"[ ℹ️  ] Created {dirName} directory")
+
+    if not os.path.isfile(os.path.join(dirName, "LICENSE.txt")):
+        shutil.copy(
+            os.path.abspath("LICENSE.txt"),
+            os.path.abspath(dirName)
+        )
+
+    if not os.path.isfile(os.path.join(dirName, "README.md")):
+        with open(os.path.join(dirName, "README.md"), "w") as readmeFile:
+            readmeFile.write(
+                f"""
+                This folder has been created by MonitoraPA on {os.path.basename(dirName)}.
+
+                The file enti.tsv has been created by AgID and distributed under CC BY 4.0.
+                An updated version can be downloaded from https://indicepa.gov.it/ipa-dati/dataset/enti
+
+                Everything else inside this folder can be used under the Hacking License.
+                See LICENCE.txt for the exact terms and conditions.
+
+                """
+            )
+
+    return dirName
+
+    # if not "check/" in argv[1]:
+    #     sys.exit("Missing GDPR compliance check.")
+
+    # if not "out/" in argv[2] or not "enti.tsv" in argv[2]:
+    #     sys.exit("Missing enti.tsv path.")
+
+    # check = os.path.splitext(os.path.basename(argv[1]))[0]
+    # outDir = os.path.dirname(argv[2])
+    # point = os.path.splitext(os.path.basename(argv[0]))[0]
+
+    # dirName = "%s/%s/%s" % (outDir, check, point)
+
+    # if not os.path.isdir(dirName):
+    #     os.makedirs(dirName, 0o755)
+    # return dirName
```

Comment:
cancella pure il codice palesemente obsoleto: se ci serve rimane nella storia di git.

Ogni tanto (raramente) quando elimino codice particolarmente intricato o che ha richiesto molto tempo per essere scritto (o perché ha richiesto studio, o perché il cliente ha cambiato idea diverse volte nel corso degli anni), lascio indietro un commento con lo SHA della ultima commit in cui era presente.
Ma in un caso come questo, non è necessario.

```suggestion
```